### ISSUE TITLE
What needs to be done in bin/i18ndude to find unstranslated strings in skins/tinymce/plugins/plonebrowser/plonebrowser.htm.pt? (Branch 1.3.x)

### ISSUE THREAD

**Issue by idgserpro** [plone/Products.TinyMCE#157]
We have some strings in

https://github.com/plone/Products.TinyMCE/blob/b99198485196b6623693283c454c6dbe3c6c640e/Products/TinyMCE/skins/tinymce/plugins/plonebrowser/plonebrowser.htm.pt#L120

and

https://github.com/plone/Products.TinyMCE/blob/b99198485196b6623693283c454c6dbe3c6c640e/Products/TinyMCE/skins/tinymce/plugins/plonebrowser/plonebrowser.htm.pt#L123

That aren't available in https://github.com/plone/Products.TinyMCE/blob/db14e8aa95938c5936cdb16efbf0a04fd0fac185/Products/TinyMCE/locales/tinymce.pot. Is this file being changed manually? If so, can the additions of `msgstr` be automatic or are we stuck on the manual approach? Is it possible to make `i18ndude` aware of `skins/tinymce/plugins/plonebrowser/plonebrowser.htm.pt`?

@mauritsvanrees

**Comment by mauritsvanrees** [comment]
This is apparently not done automatically.  These commands would update the translation files, and result in those string being picked up:

```
i18ndude rebuild-pot --pot locales/tinymce.pot --create tinymce --exclude "*py" .
i18ndude rebuild-pot --pot locales/plone.tinymce.pot --create plone.tinymce --merge locales/plone.tinymce-manual.pot .
i18ndude sync --pot locales/tinymce.pot $(find . -name 'tinymce.po')
i18ndude sync --pot locales/plone.tinymce.pot $(find . -name 'plone.tinymce.po')
```

But then in most `tinymce.po` files about 30 entries are added and about 600 removed. I guess a lot of those 600 *are* needed, but are not picked up by i18ndude.  So that is not workable at the moment.

Note that Products.TinyMCE is not used in Plone 5 anymore (the tinymce editor lives in `mockup` I think), so if we change anything here, it will be a lot of work for only Plone 4.

**Comment by idgserpro** [comment]
@mauritsvanrees thanks for your fast response.

If I understand correctly, it can be fixed, but the work is no feasible since Products.TinyMCE is not used in Plone 5 anymore ([only 1.4.x for archetypes](http://dist.plone.org/release/5.1-latest/versions.cfg)): If someone wants this somehow fixed in 1.3.x, they're on their own providing a PR that fixes it.

Another option is to keep it manual and provide specific PR's adding strings on demand.

If we're in agreement above, this issue can be closed.

**Comment by mauritsvanrees** [comment]
Yes, that is the idea.
